p2p/discover/table.go

```diff
@@ -50,8 +50,8 @@ const (
 	bucketMinDistance = hashBits - nBuckets // Log distance of closest bucket
 
 	// IP address limits.
-	bucketIPLimit, bucketSubnet = 2, 24 // at most 2 addresses from the same /24
-	tableIPLimit, tableSubnet   = 10, 24
+	bucketIPLimit, bucketSubnet = 2 * common.NumRegionsInPrime * common.NumZonesInRegion, 24  // Up to 2x num chains in network
+	tableIPLimit, tableSubnet   = 10 * common.NumRegionsInPrime * common.NumZonesInRegion, 24 // Up to 10x num chains in network
```

Comment:
For these, if each node is running 13 clients each with their own enode, to scale linearly we should just multiply by 13 right? Currently we multiply by 9

Reply:
fixed